Source/NSTreeController.m

```diff
@@ -80,13 +102,21 @@ - (void) dealloc
   RELEASE(_countKeyPath);
   RELEASE(_leafKeyPath);
   RELEASE(_sortDescriptors);
+  RELEASE(_arranged_objects);
+
```

Comment:
Shouldn't we release `_selection_index_paths` here as well?

Reply:
Done